vulnerability/events_creator.go

```diff
@@ -193,3 +193,10 @@ func getDataSource(vul trivy_types.DetectedVulnerability) DataSource {
 		URL:  vul.DataSource.URL,
 	}
 }
+
+func getPublishedDate(vul trivy_types.DetectedVulnerability) time.Time {
+	if vul.PublishedDate == nil {
```

Comment:
The default vaule of `time.Time` is `0001-01-01 00:00:00 +0000 UTC`. I would consider `time.Now()`

Reply:
IMO, we can omit this field in case it does not have a value.